app/Models/DuesTransactionMerchandise.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\Pivot;
+
+class DuesTransactionMerchandise extends Pivot
+{
+    /**
+     * The attributes that should be mutated to dates.
+     *
+     * @var array<string>
+     */
+    protected $casts = [
+        'provided_at' => 'datetime',
+    ];
+
+    /**
+     * Indicates if the IDs are auto-incrementing.
+     *
+     * @var bool
+     */
+    public $incrementing = true;
```

Comment:
🤔

Reply:
Per Laravel docs this is required for pivot models if they have an autoincrementing ID column.